src/otx/core/data/module.py

```diff
@@ -34,18 +35,26 @@ def __init__(self, task: OTXTaskType, config: DataModuleConfig) -> None:
         self.subsets: dict[str, OTXDataset] = {}
         self.save_hyperparameters()
 
-        dataset = DmDataset.import_from(
-            self.config.data_root,
-            format=self.config.data_format,
-        )
+        if self.config.data_format == "common_semantic_segmentation":
+            # temporary hack, should be fixed later
```

Comment:
So, if the `common_semantic_segmentation` data format can't be imported by using the root path ?

Reply:
Yes, It can't be imported by Datumaro with just a root path.